src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerTelemetry.cs

```diff
@@ -89,6 +89,13 @@ public sealed class AnalyzerTelemetryInfo
         /// </summary>
         public int OperationBlockActionsCount { get; set; } = 0;
 
+        /// <summary>
+        /// Count of registered suppression actions.
+        /// This is the same as count of <see cref="DiagnosticSuppressor"/>s as each suppressor
+        /// has a single suppression action, i.e. <see cref="DiagnosticSuppressor.ReportSuppressions(SuppressionAnalysisContext)"/>.
+        /// </summary>
+        public int SuppressionActionsCount { get; set; } = 0;
```

Comment:
Is anyone actually using this stuff?

Reply:
Yes, it is logged as telemetry in IDE analyzer execution as well as reported by AnalyzerRunner.